### ISSUE TITLE
Edit contacts

### ISSUE THREAD

**Issue by marcobischoff** [hql287/Manta#212]
Please answer these question so I can help you faster.

**Is it a bug or a feature request?**

- [ ] It's a bug.
- [ x ] I'm requesting a new feature.

**Which version are you using?**

- [ x ] macOS
- [ ] Windows
- [ ] Linux

**Which is the version/distribution of the OS?**
macOS High Sierra

**Describe your issue**
It would be nice to have the option to edit the contact information, as it could be that the phone number changes.

**Comment by hql287** [comment]
@marcobischoff Yes, this would be nice. 👍Would you like to work on a PR for this? I think the implementation is straightforward and very similar to invoices C.R.U.D operations. 

A couple of things to keep in mind:

1. Once the contact information is changed, would you change the invoices that are associated with that contact?
2. While you're at it, probably tackle these as well https://github.com/hql287/Manta/issues/178,  https://github.com/hql287/Manta/issues/58 :)

**Comment by marcobischoff** [comment]
@hql287 I'm totaly new to react/electron, but I can have a look and maybe I can create a PR for this.

**Comment by hql287** [comment]
Well, then I would recommend picking up small issues to get familiar with the code base first. May I suggest:
*  https://github.com/hql287/Manta/issues/29 
* https://github.com/hql287/Manta/issues/129
* https://github.com/hql287/Manta/issues/86
* https://github.com/hql287/Manta/issues/76
